front-packages/akeneo-design-system/src/icons/ExternalLinkIcon.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import {IconProps} from './IconProps';
+
+const ExternalLinkIcon = ({title, size = 24, color = 'currentColor', ...props}: IconProps) => (
+    <svg viewBox='0 0 24 24' width={size} height={size} {...props}>
+        {title && <title>{title}</title>}
+        <path
+            d='M16.047 3.02258H20.9561L20.9561 7.93171M20.9561 3.02258L12.2914 11.6874L20.9561 3.02258Z'
```

Comment:
double quotes instead of single in all XML attributes to be consistent with other components of this directory

```suggestion
            d="M16.047 3.02258H20.9561L20.9561 7.93171M20.9561 3.02258L12.2914 11.6874L20.9561 3.02258Z"
```